spec/controllers/front_end_builds/apps_controller_spec.rb

```diff
@@ -113,12 +113,12 @@ module FrontEndBuilds
 
         context 'the response' do
           subject { response }
-          it { should be_success }
+          it { expect(subject.successful?).to be true }
         end
 
         context 'the data' do
           subject { json['app']['id'] }
-          it { should_not be_nil }
+          it { expect(subject).not_to be nil }
```

Comment:
We can eliminate the negation by testing for presence or truthiness: `expect(subject).to be_truthy`

Reply:
✅